From 9bd87983e123f31addf41b76207cb844935a0d57 Mon Sep 17 00:00:00 2001
From: "R. Adi Permana" <romadhonadipermana279f@gmail.com>
Date: Wed, 23 Sep 2020 01:04:58 +0700
Subject: [PATCH 56/99] ARM64: configs: ulysse: Disable unused power drivers

We only use bq2560x-charger for charger driver & bq27426-fg as fuel gauge
---
 arch/arm64/configs/ulysse_defconfig | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/arch/arm64/configs/ulysse_defconfig b/arch/arm64/configs/ulysse_defconfig
index a7e17d878c4f..4098d39b8c55 100644
--- a/arch/arm64/configs/ulysse_defconfig
+++ b/arch/arm64/configs/ulysse_defconfig
@@ -361,16 +361,16 @@ CONFIG_GPIO_SYSFS=y
 CONFIG_GPIO_QPNP_PIN=y
 CONFIG_POWER_RESET_QCOM=y
 CONFIG_QCOM_DLOAD_MODE=y
-CONFIG_QPNP_FG=y
-CONFIG_SMB135X_CHARGER=y
-CONFIG_SMB1360_CHARGER_FG=y
-CONFIG_SMB1355_SLAVE_CHARGER=y
-CONFIG_SMB1351_USB_CHARGER=y
-CONFIG_QPNP_SMB5=y
-CONFIG_QPNP_SMBCHARGER=y
-CONFIG_QPNP_VM_BMS=y
-CONFIG_QPNP_LINEAR_CHARGER=y
-CONFIG_QPNP_TYPEC=y
+# CONFIG_QPNP_FG is not set
+# CONFIG_SMB135X_CHARGER is not set
+# CONFIG_SMB1360_CHARGER_FG is not set
+# CONFIG_SMB1355_SLAVE_CHARGER is not set
+# CONFIG_SMB1351_USB_CHARGER is not set
+# CONFIG_QPNP_SMB5 is not set
+# CONFIG_QPNP_SMBCHARGER is not set
+# CONFIG_QPNP_VM_BMS is not set
+# CONFIG_QPNP_LINEAR_CHARGER is not set
+# CONFIG_QPNP_TYPEC is not set
 CONFIG_QPNP_QG=y
 CONFIG_MSM_APM=y
 CONFIG_SENSORS_QPNP_ADC_VOLTAGE=y
-- 
2.20.1

